src/common/transformations/src/transformations/low_precision/mark_dequantization_subgraph.cpp

```diff
@@ -107,9 +107,6 @@ ov::pass::MarkDequantizationSubgraph::MarkDequantizationSubgraph(const element::
                 ov::disable_constant_folding(scale);
                 ov::unmark_as_decompression(scale);
                 ov::enable_keep_const_precision(scale->get_input_node_shared_ptr(0));
-            } else {
-                ov::enable_constant_folding(scale);
-                ov::disable_keep_const_precision(scale->get_input_node_shared_ptr(0));
```

Comment:
Shouldn't scale be constant foldable if `fold_multiply_const` is true?

Reply:
Renamed param to `disable_fold_multiply_const` in order to have more clear logic